app/Services/NotificationService.php

```
@@ -0,0 +1,290 @@
+     * @return array
+     */
+    private function userShortcodes($user = null)
+    {
+        return [
+            'username' => $user->username ?? '',
+            'email' => $user->email ?? '',
+            'phone' => $user->phone ?? '',
+            'name' => $user->name ?? '',
+            'role' => $user->role ?? '',
+        ];
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'userShortcodes'. (Unused Code Rules)

(UnusedPrivateMethod)